src/main/java/lotto/service/LottoManager.java

```diff
@@ -0,0 +1,38 @@
+package lotto.service;
+
+import lotto.domain.BonusNumber;
+import lotto.domain.Lotto;
+import lotto.domain.Lottos;
+import lotto.domain.PurchasingMoney;
+import lotto.domain.generator.IssuedLottoGenerator;
+import lotto.domain.generator.WinningLottoGenerator;
+import lotto.domain.result.ProfitRate;
+import lotto.domain.result.WinningDetail;
+import lotto.util.Input;
+import lotto.util.Output;
+
+public class LottoManager implements Manager {
+
+    @Override
+    public void run() {
+        PurchasingMoney purchasingMoney = PurchasingMoney.from(Input.inputPurchasingMoney());
+        Lottos issuedLotto = purchaseLotto(purchasingMoney);
```

Comment:
Lotto"s"라고 해주는게 가독성에 더 좋지 않을까요?
```suggestion
        Lottos issuedLottos = purchaseLotto(purchasingMoney);
```